DrawAction/DrawText.swift

```diff
@@ -49,11 +49,11 @@ final public class DrawText : DrawAction {
         paragraphStyle.lineBreakMode = lineBreakMode
         paragraphStyle.alignment = alignment
         
-        let attributes: [String : AnyObject] = [
-            NSFontAttributeName : font,
-            NSParagraphStyleAttributeName: paragraphStyle,
-            NSForegroundColorAttributeName: color,
-            NSUnderlineStyleAttributeName: underlineStyle.rawValue as AnyObject
+        let attributes: [NSAttributedString.Key : AnyObject] = [
```

Comment:
Minor, but you can get rid of the full NSAttributedString.Key prefixes. .font, .paragraphStyle, etc. will work.

Reply:
Done